SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -198,7 +199,7 @@ namespace SpatialConstants
 	const WorkerRequirementSet ClientOrServerPermission{ {UnrealClientAttributeSet, UnrealServerAttributeSet} };
 
 	static const FString ClientsStayConnectedURLOption = TEXT("clientsStayConnected");
-	static const FString SnapshotURLOption = TEXT("snapshot=");
+	static const FString SessionIdURLOption = TEXT("sessionId=");
```

Comment:
Hmm, potentially this should be `spatialSessionId`?  I can see `sessionsId` being something a developer would use for their own information.

Reply:
✓